server/model.js

```diff
@@ -0,0 +1,29 @@
+const { Review } = require('../database');
+
+// MySQL + Sequalize
+const model = {
+  mysqlSequalize: {
+    // GET
+    findAll: (itemId) => Review.findAll({
+      where: {
+        ItemId: itemId,
+      },
+    }),
+    // POST
+    create: (newReview) => Review.create(newReview),
+    // DELETE
+    destroy: (reviewId) => Review.destroy({
+      where: { id: reviewId },
+    }),
+    // UPDATE
+    update: (updatedFields, reviewId) => Review.update(updatedFields, {
+      where: { id: reviewId },
+    }),
+  },
+
+  mongoDb: {
```

Comment:
It looks like you're using Postgres and CouchDB - is this MongoDB line from the legacy code? Is it needed?